src/install.py

```diff
@@ -251,10 +251,11 @@ def applyKickstart(self):
                 self.nfsdir_entry.set_text(self.ks.method.dir)
 
         elif self.ks.method.method == "url":
-            tokens = string.split(self.ks.method.url, "://")
+            if self.ks.method.url is not None:
+                tokens = string.split(self.ks.method.url, "://")
 
-            protocol = tokens[0]
-            data = tokens[1]
+                protocol = tokens[0]
+                data = tokens[1]
 
             if protocol == "ftp":
```

Comment:
If `self.ks.method.url` is indeed `None`, this line would throw a `NameError: undefined 'protocol'`.

Reply:
```python
    elif self.ks.method.method == "url":
        protocol = ""
        data = ""

        if self.ks.method.url is not None:
            tokens = string.split(self.ks.method.url, "://")
            protocol = tokens[0]
            data = tokens[1]

        if protocol == "ftp":
            ...
        if protocol == "http":
            ...
```

That's how I did it locally, and it's working well - cut it down in the PR for brevity, but that was an oversight. You're right, good catch!

Do you think this should be alright, or am I still missing something?